incremental-copy-docs/README.md

```diff
@@ -37,7 +52,9 @@ To quickly deploy the Migration tool, you can use the Azure deployment below. Pl
 
 		[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fapiforfhir-migration-tool%2Fmain%2Finfra%2Fmain.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fapiforfhir-migration-tool%2Fmain%2Finfra%2FuiDefForm.json)
 
-    2. Or, deploy the migration tool manually
+    **NOTE** : To avoid azure resource already deployed with the same name use unique Prefix for FHIR Migration Tool resources during deployment.
```

Comment:
Capitalization: "azure" should be "Azure".

Reply:
Done with the changes.